lib/taurus/qt/qtgui/panel/test/test_taurusvalue.py

```diff
@@ -51,9 +51,8 @@ def test_bug126(self):
         '''Verify that case is not lost when customizing a label (bug#126)'''
         w = self._widget
         # self._widget.setModel('eval:1')
-        self._widget.setModel('tango:' + DEV_NAME + '/double_scalar')
+        self._widget.setModel('tango:' + DEV_NAME + '/MIXEDcase')
```

Comment:
I may be wrong, but I think that the changes that you are proposing in this test are not ok.

The original purpose of this test was to check for bug sdn-SF#126 (corresponding to #53 ): we want to make sure that the case is kept when *setting* the label.

With your proposed changes, we are checking that a mixed label is not lost, which IMHO is a good thing to check, but in a different test.

If you agree, can you please implement both tests? (and in the meanwhile, it would be nice to change the name and docstring to use `#53` instead of `#126`)

Reply:
I agree. I had to add a new features to the wiedgets:
* TaurusLabel - setPermanentText
* TaurusValue - setLabelText

I have also implemented backwards compatibility to the setLabelConfig so the original test_bug126 works.